src/styles/stars.scss

```diff
@@ -0,0 +1,24 @@
+.stars {
+  display: flex;
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    margin-right: 4px;
+
+    background-image: url(../images/star.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+  }
+
+  &__star:last-child {
+    margin-right: 0;
+  }
+  &.stars--1 .stars__star:nth-of-type(-n + 1),
+  &.stars--2 .stars__star:nth-of-type(-n + 2),
+  &.stars--3 .stars__star:nth-of-type(-n + 3),
+  &.stars--4 .stars__star:nth-of-type(-n + 4),
+  &.stars--5 .stars__star:nth-of-type(-n + 5) {
```

Comment:
According to the [Code Style Rules](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), combinators should be surrounded by spaces. Please add spaces around the '+' in the 'nth-of-type' pseudo-class.